styles/portfolio.css

```diff
@@ -0,0 +1,83 @@
+.portfolio__tittle {
+  width: 100%;
+  height: 38px;
+  font-family: "Raleway", sans-serif;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 38px;
+  text-align: center;
+  letter-spacing: 0.02em;
+  color: #222222;
+}
+.portfolio__container {
+  display: grid;
+  grid-template-columns: repeat(25, 1fr);
+  grid-template-rows: 1fr 1fr;
+  width: 100%;
+  /* max-width: 1440px; */
```

Comment:
Usuńmy komentarze :D

Reply:
usuwam... ten komentarz był stąd, że nie byłem pewien czy width w Figmie jest 1440px , bo taka jest szerokość frame`a desktop czy taka ma być szerokość sekcji no i to było tak do ewentualnego włączenia :)